app/grandchallenge/algorithms/views.py

```diff
@@ -640,6 +639,22 @@ class JobsList(PaginatedTableListView):
         "inputs__image__files__file",
         "comment",
     ]
+
+    columns = [
+        Column(title="Detail"),
+        Column(title="Created", sort_field="created"),
+        Column(title="Creator", sort_field="creator__username"),
+        Column(title="Status", sort_field="status"),
+        Column(title="Visibility", sort_field="public"),
+        Column(
+            title="Comment",
+            sort_field="comment",
+            optional_condition=lambda obj: bool(obj.comment),
+        ),
+        Column(title="Results", sort_field="outputs"),
```

Comment:
Have you tried this? I don't think ordering on a many to many field like this works.

Reply:
Good point. Not much use having this here.

Reply:
It doesn't! A such I have removed it!